### DISCUSSION TITLE
aruco header not found in OpenCV

### DISCUSSION THREAD

**Discussion by ZA0068** [microsoft/vcpkg#23291] [General]
I have installed vcpkg OpenCV in my system.

But when I invoke #include <opencv2/aruco.hpp>

I get an error where the file does not exist.

When I checked the installation location of OpenCV. I can't find anything related to aruco.

the official OpenCV does have this header, who does VCPKG not?

**Comment by vicroms** [comment]
It looks like aruco is part of opencv_contrib.
Try `vcpkg install opencv[contrib]`.

**Reply by 45739847** [reply]
thanks, it helped me with my problem
